scripts/start-deployCF

```diff
@@ -35,10 +35,38 @@ FTB_SERVER_MOD=${FTB_SERVER_MOD:-$CF_SERVER_MOD}
 log "Looking for Feed-The-Beast / CurseForge server modpack."
 requireVar FTB_SERVER_MOD
 
+downloadModpack() {
+  srv_modpack=${FTB_SERVER_MOD}
+  if isURL "${srv_modpack}"; then
+    log "Downloading modpack from ${srv_modpack}..."
+    if ! srv_modpack=$(get -o /data --output-filename --skip-existing "${srv_modpack}"); then
+      log "ERROR: failed to download modpack"
+      exit 1
+    fi
+  fi
+  if [[ "${srv_modpack:0:5}" == "data/" ]]; then
+      # Prepend with "/"
+      srv_modpack="/${srv_modpack}"
+  fi
+  if [[ ! "${srv_modpack:0:1}" == "/" ]]; then
+      # If not an absolute path, assume file is in "/data"
+      srv_modpack=/data/${srv_modpack}
+  fi
+  if [[ ! -f "${srv_modpack}" ]]; then
+      log "FTB server modpack ${srv_modpack} not found."
+      exit 2
+  fi
+  if [[ ! "${srv_modpack: -4}" == ".zip" ]]; then
+      log "FTB server modpack ${srv_modpack} is not a zip archive."
+      log "Please set FTB_SERVER_MOD to a file with a .zip extension."
+      exit 2
+  fi
+  FTB_SERVER_MOD=${srv_modpack}
+}
+
 if ! isTrue "${USE_MODPACK_START_SCRIPT:-true}"; then
   if ! [ -f "${FTB_SERVER_MOD}" ]; then
-    log "ERROR unable to find requested modpack file ${FTB_SERVER_MOD}"
-    exit 2
+    downloadModpack
   fi
```

Comment:
I'm thinking the conditional can be taken away and this if-block just replaced with a call to `downloadMockpack`. That function is doing URL and file-existence checks.

Reply:
Yeah you're right. Just tested it with a mounted volume zip, still works